src/improvements/script/create-template.sh

```diff
@@ -9,8 +9,64 @@ create_template() {
             read -r filename
         fi
         if [[ "$filename" == *.sol ]]; then
-            template_path="template/$filename"
-            touch "$template_path"
+            contract_name="${filename%.sol}"
+            template_path="templates/$filename"
```

Comment:
This breaks the `just new template` command. 
```suggestion
            template_path="template/$filename"
```